test/e2e/app-dir/app-routes/app-custom-routes.test.ts

```diff
@@ -1,18 +1,35 @@
 import { createNextDescribe } from 'e2e-utils'
+import fs from 'fs-extra'
+import { check } from 'next-test-utils'
+import path from 'path'
+import { Readable } from 'stream'
+
 import {
   withRequestMeta,
   getRequestMeta,
   cookieWithRequestMeta,
 } from './helpers'
-import { Readable } from 'stream'
-import { check } from 'next-test-utils'
 
 createNextDescribe(
   'app-custom-routes',
   {
     files: __dirname,
   },
-  ({ next, isNextDev, isNextStart }) => {
+  ({ next, isNextDeploy, isNextDev, isNextStart }) => {
+    beforeAll(async () => {
+      if (isNextDev) {
+        await fs.move(
+          path.join(next.testDir, 'app/_lowercase'),
+          path.join(next.testDir, 'app/lowercase'),
+          { overwrite: true }
+        )
+        await fs.move(
+          path.join(next.testDir, 'app/_default'),
+          path.join(next.testDir, 'app/default'),
+          { overwrite: true }
+        )
+      }
+    })
```

Comment:
You'll need to move these back in `afterAll`

Reply:
I thought `next.testDir` just kinda self-destructs after tests are done? I also have problems trying to move those folders back - they don't exist anymore when my `afterAll` is called it  seems.

Reply:
I think you're right!